erpnext/projects/doctype/project/project.py

```diff
@@ -394,12 +394,6 @@ def handle_task_linking(new_task_list):
 			for item in new_task_list:
 				if old_task.parent_task == item['previous_task_name']:
 					new_task.parent_task = item['new_task_name']
-		if old_task.depends_on:
```

Comment:
- The Project duplication works properly even before the fix. Is there any specific scenario or edge case which this patch fixes ?

Reply:
@hrwX  this chunks of code trying to append new task to old parent task like

task A  ---> task A1
task B child of A   ---> task B1 child of A1
and above code is also appending to old task like

task B1 child of A

Reply:
![dup_task](https://user-images.githubusercontent.com/58166671/93879408-48673100-fcf9-11ea-9d2b-7ee10e50dad3.gif)

The new Project's dependent tasks was having the same dependent task as previous Project(as shown in gif).
so to change it to newer task we applied this logic which was failing again. where now this time both Projects were having new task as dependent task.
To stop applying this logic on older Project tasks, we have removed this part of code which was previously updating dependent tasks